code/modules/mob/living/carbon/human/death.dm

```diff
@@ -8,15 +8,21 @@
 		if(GIB_TYPE_ROBOTIC)
 			new /obj/effect/temp_visual/gib_animation(loc, "gibbed-r")
 
-/mob/living/carbon/human/dust_animation()
-	if(!dna)
-		new /obj/effect/temp_visual/dust_animation(loc, "dust-h")
-		return
-	switch(dna.species.species_gibs)
-		if(GIB_TYPE_HUMAN)
-			new /obj/effect/temp_visual/dust_animation(loc, "dust-h")
-		if(GIB_TYPE_ROBOTIC)
-			new /obj/effect/temp_visual/dust_animation(loc, "dust-r")
+/mob/living/carbon/human/dust(just_ash, drop_items, force)
```

Comment:
this entire method is identical to `/mob/living/dust()` is it not? just remove the override